common/src/main/java/org/apache/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -86,9 +112,15 @@ public String toString()
            ", connectURI='" + getConnectURI() + '\'' +
            ", user='" + user + '\'' +
            ", passwordProvider=" + passwordProvider +
+           ", dbcpPropertiesFile=" + dbcpPropertiesFile +
            '}';
   }
 
+  String getDbcpPropertiesFile()
```

Comment:
Any reason this cannot be `public`?

Reply:
It is package private since it is only used for testing.  No reason to expose the file path, just the properties.